app/src/main/java/com/example/drawingactivity/ColorUtils.kt

```diff
@@ -0,0 +1,94 @@
+package com.example.drawingactivity
+
+import android.graphics.Color
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+object ColorUtils {
+
+    private val HEXADECIMAL_PATTERN: Pattern = Pattern.compile("\\p{XDigit}+")
```

Comment:
I saw this same strategy online but I was unsure how well it would work, how did this feel for you?